packages/git_lfs.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Git_lfs < Package
+  description 'Git extension for versioning large files'
+  homepage 'https://git-lfs.github.com/'
+  version '2.2.1'
+  source_url 'https://github.com/git-lfs/git-lfs/archive/v2.2.1.tar.gz'
+  source_sha256 'fede2b31b0539fd4a580f831867caac1b5d5dc4405e938c4ee0bfeacfb78ad7a'
+
+  depends_on 'go'
+
+  def self.build
```

Comment:
Yes. Go uses /tmp by default and we don't have exec permission on /var.  It appears like below.
```
$ crew install git_lfs
git_lfs: Git extension for versioning large files
https://git-lfs.github.com/
version 2.2.1
No precompiled binary available for your platform, downloading source...
######################################################################## 100.0%
Archive downloaded
Unpacking archive, this may take a while...
gcc (GCC) 4.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
fork/exec /tmp/go-build400317567/command-line-arguments/_obj/exe/build: permission denied
git_lfs failed to install: `script/bootstrap` exited with 1
```